.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "flutter-survey",
+            "request": "launch",
+            "type": "dart"
+        },
+        {
+            "name": "flutter-survey (profile mode)",
+            "request": "launch",
+            "type": "dart",
+            "flutterMode": "profile"
+        }
+    ]
+}
```

Comment:
eof

Reply:
this will be easily fixed in the following PRs, I will give it a go 😛